src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCacheOptions.cs

```diff
@@ -42,6 +42,9 @@ public class MemoryCacheOptions : IOptions<MemoryCacheOptions>
             }
         }
 
+        [System.Obsolete("This is obsolete and will be removed in a future version.")]
```

Comment:
In general, is this the right message? Maybe instead:

"Present for backward compatibility only, please do not use in new code"

?

Reply:
None of these messages have been scrubbed - I just brought back them as they were in the last release or put a TODO in place.  This PR isn't meant to review for style/final merge.  It's sharing an outline to drive discussion on next steps.